ci/recipe/meta.yaml

```diff
@@ -19,7 +19,7 @@ requirements:
 
   run:
     - python {{ python }}
-    - vsearch 2.7.0
+    - vsearch 2.13.7
```

Comment:
This is going to require a bit of coordination with q2-deblur, q2-feature-classifier, and q2-quality-control, which all specify `vsearch <=2.7.0`.